etc/cvmfs/domain.d/osgstorage.org.conf

```diff
@@ -13,27 +13,28 @@ CVMFS_KEYS_DIR=$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys/osgstora
 
 # subset of OSDF caches used by OSG
 # North America
-CVMFS_EXTERNAL_URL="http://stashcache.t2.ucsd.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://dtn-pas.jack.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://unl-cache.nationalresearchplatform.org:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-chicago-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-new-york-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-stash-sfu-computecanada-ca.nationalresearchplatform.org:8000/"
+CVMFS_EXTERNAL_URL="http://stashcache.t2.ucsd.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://dtn-pas.jack.nrp.internet2.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://unl-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-chicago-stashcache.nrp.internet2.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-new-york-stashcache.nrp.internet2.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-stash-sfu-computecanada-ca.nationalresearchplatform.org:8443/"
```

Comment:
This one is supposed to stay on port 8000.  It shows up as a diff compared to master, and only port 8000 works in my test.

Reply:
yes, fixed.